src/api.js

```diff
@@ -141,6 +141,7 @@ var api = {
     utils.envOverride("FIREBASE_GOOGLE_URL", "https://www.googleapis.com")
   ),
   hostingOrigin: utils.envOverride("FIREBASE_HOSTING_URL", "https://web.app"),
+  identityOrigin: "https://identitytoolkit.googleapis.com",
```

Comment:
wrap this in `utils.envOverride`, maybe with `FIREBASE_IDENTITY_URL`?

```suggestion
  identityOrigin: utils.envOverride("FIREBASE_IDENTITY_URL", "https://identitytoolkit.googleapis.com"),
```